WordPress/Tests/Arrays/ArrayDeclarationSpacingUnitTest.inc

```diff
@@ -2,7 +2,23 @@
 
 $good = array( 'value1', 'value2' ); // Ok.
 
-$query_vars = array('post_type' => 'food'); // Bad, no spaces after opening and before closing parenthesis.
+$query_vars = array('food'); // Bad, no spaces after opening and before closing parenthesis.
 
 // Test for fixing of extra whitespace.
 $test = array(   1, 2     );
+
+$bad = array( 'key' => 'value' ); // Bad, each value of an associative array should start on a new line.
```

Comment:
Since this is what the handbook says, I guess lets go with it for now. If I feel the need in the future, I'll open a PR adding the option to ignore single-element arrays.